### PR TITLE
Document and format

### PR DESCRIPTION
**Pull request by JordanRushing**
These changes pass the provided tests but README.md instructions for running the bot locally and via Puma do not seem to work. I can update the docs or my local config to allow for functional testing beyond Rspec

### Passing Tests:
```
$ bundle exec rake

. . .

Finished in 0.0263 seconds (files took 0.63336 seconds to load)
7 examples, 0 failures


COVERAGE:  66.90% -- 95/142 lines in 7 files

+----------+-----------------------------------+-------+--------+--------------------------------------+
| coverage | file                              | lines | missed | missing                              |
+----------+-----------------------------------+-------+--------+--------------------------------------+
|  57.58%  | lib/chillbud/chillbud.rb          | 33    | 14     | 43-46, 48, 57-58, 63, 65-68, 74, 79  |
|  62.96%  | lib/chillbud/jobs/chillbud_job.rb | 27    | 10     | 18, 24-25, 27-29, 33-36              |
|  65.79%  | lib/chillbud/plugins/reminder.rb  | 38    | 13     | 20, 25, 47, 49, 51, 56, 67-71, 75-76 |
|  73.68%  | app.rb                            | 19    | 5      | 33-36, 40                            |
|  75.00%  | lib/chillbud/plugins/hello.rb     | 8     | 2      | 8, 12                                |
|  76.92%  | lib/chillbud/plugins/plugin.rb    | 13    | 3      | 21-23                                |
+----------+-----------------------------------+-------+--------+--------------------------------------+
1 file(s) with 100% coverage not shown
```
### Failure to run:
```
ubuntu at ip-172-31-16-223 in ~/chillbud (document-and-format)
$ chillbud
Traceback (most recent call last):
        2: from /home/ubuntu/.rvm/gems/ruby-2.5.1/bin/chillbud:23:in `<main>'
        1: from /usr/share/rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/usr/share/rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem chillb
ud (>= 0.a) with executable chillbud (Gem::GemNotFoundException)
```

## Revision 2
After second revision all tests pass and the bot runs successfully:

```
JordanToday at 4:32 PM
!reminder 10s @kbougy is a goober
ChillbudBOTToday at 4:32 PM
Saved your reminder: @Jordan. You will be reminded at 2018-12-18 22:32:07
2018-12-18 22:32:07: @kbougy is a goober @Jordan
```

### CODE REVIEW

**Comment by kbougy** [comment]
Try `bundle exec chillbud` ;)

**Comment by kbougy** [comment]
Gotta get some functional/integration tests running to verify that the server at least gets the right stuff and the examples in the readme work. Shouldn't be too hard to do. I added a task in trello for it.

File: README.md [hunk 1]
```diff
@@ -27,15 +27,13 @@ Start the server
...
+
 Stop the bot
 
-    $ curl -d "{\"discord_token\":\"yourtoken\"" \ 
-           -h "content-type: application/json" \
-           -x post http://localhost:80/api/v1/stop 
+    curl -d "discord_token=$DISCORD_TOKEN" -X POST \
+         http://localhost:3000/api/v1/stop
            
 ## Development
```
Comment (kbougy) [inline README.md]: Add some steps instructing the user on how to use the formatting stuff you added.

**Review by kyrivanderpoel — CHANGES_REQUESTED** [review]
Only one comment that needs to be addressed :)

**Review by kyrivanderpoel — COMMENTED** [review]
Looks good to me!

**Review by kyrivanderpoel — APPROVED** [review]

**Pull request merged**